decidim-core/lib/decidim/core.rb

```diff
@@ -845,6 +845,12 @@ def self.organization_settings(model)
     0.seconds
   end
 
+  # Public Setting that allows to configure which component will have "Use my current location" button
+  # in a geocoded address field. Accepts an array of component manifest names
+  config_accessor :show_my_location_button do
+    [:proposals]
```

Comment:
As this can be useful for implementator for configurations, let's add also a line in 
`decidim-generators/lib/decidim/generators/app_templates/initiatlizer.rb` like this:

` config.show_my_location_button = Rails.application.secrets.decidim[:show_my_location_button].present?`

And in secrets.yml.erb:
`show_my_location_button <%%= Decidim::Env.new("DECIDIM_SHOW_MY_LOCATION_BUTTON").to_boolean_string %>`

Reply:
@microstudi if i would like to add the button in the accountability, i only need to add it here? or i have to patch the _form as well ?

Reply:
yes, btw I thing they should be a `:any` or `:all` configuration that ignores the manifest of the component. Making notes on a review now

Reply:
the version with `[:all]` implemented